GameData/ROCapsules/PartConfigs/GeminiBDB/GeminiHS.cfg

```diff
@@ -0,0 +1,74 @@
+PART
+{
+	name = ROC-GeminiHSBDB
+	module = Part
+	author = CobaltWolf, capkirk
+
+	RSSROConfig = True
+
+	MODEL
+	{
+		model = ROCapsules/Assets/BDB/Gemini/bluedog_Gemini_Heatshield_1p5m
+		rotation = 0, 180, 0
+	}
+
+	rescaleFactor = 1.524
+	node_stack_bottom = 0.0, 0.075149, 0.0, 0.0, -1.0, 0.0, 1
+	node_stack_top = 0.0, 0.075149, 0.0, 0.0, 1.0, 0.0, 1
+	CoPOffset = 0.0, 1.0, 0.0
+	CoLOffset = 0.0, -0.15, 0.0
+
+	TechRequired = flightControl
+	entryCost = 1000
+	cost = 360
+	category = Thermal
+	subcategory = 0
+	title = Gemini Heatshield
+	manufacturer = McDonnell Aircraft
+	description = Heatshield for the Gemini Capsule. Can be switched to the Gemini B configuration with a hatch
+	attachRules = 1,0,1,0,0
+	mass = 0.050
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 72
+	breakingTorque = 72
+	crashTolerance = 18
+	maxTemp = 1500
+	skinMaxTemp = 2600
+	skinSkinConductionMult = 0.1
+	fuelCrossFeed = False
```

Comment:
Note that maxTemp, skinMaxTemp, and mass will be set because of the heatshield tags below.
Unsure if you really want skinSkinConductionMult to be 0.1.  This would help the shield exposed temp heat more/faster versus conducting some of that heat into the unexposed part of the skin... but shields probably have very little unexposed portions of skin (typically it'd be the back side, but they're usually sized close to the diameter of the part they protect.)